pytest_fixtures/component/provision_pxe.py

```diff
@@ -55,24 +56,26 @@ def module_provisioning_rhel_content(
             reposet=constants.REPOS['kickstart'][name]['reposet'],
             releasever=constants.REPOS['kickstart'][name]['version'],
         )
-
-        rh_repo_id = enable_rhrepo_and_fetchid(
-            basearch=constants.DEFAULT_ARCHITECTURE,
-            org_id=module_sca_manifest_org.id,
-            product=constants.REPOS[name]['product'],
-            repo=constants.REPOS[name]['name'],
-            reposet=constants.REPOS[name]['reposet'],
-            releasever=constants.REPOS[name]['releasever'],
-        )
+        # do not sync content repos for discovery based provisioning.
+        if not module_provisioning_sat.provisioning_type == 'discovery':
+            rh_repo_id = enable_rhrepo_and_fetchid(
```

Comment:
This must be accessed through satellite object `module_provisioning_sat.enable_rhrepo_and_fetchid`

Reply:
non-blocking, this can be updated in master first and then we can cpick this to 6.12.z, so merging this PR